docs/fundamentals/toc.yml

```diff
@@ -92,69 +92,69 @@ items:
     - name: Use Visual Studio
       items:
       - name: Create a console app
-        href: ../core/tutorials/with-visual-studio.md
         displayName: tutorials, visual studio, vs
+        href: ../core/tutorials/with-visual-studio.md
```

Comment:
What's the reason for ordering name-displayname-href instead of name-href-displayname?

Reply:
The archive tool made all these changes automatically. I'll revert them.